src/k8s/pkg/k8sd/app/app.go

```diff
@@ -55,10 +55,10 @@ func New(ctx context.Context, cfg Config) (*App, error) {
 func (a *App) Run(customHooks *config.Hooks) error {
 	// TODO: consider improving API for overriding hooks.
```

Comment:
nit: is this still relevant? Seems outdated.

Reply:
yeah, we now pass a `*config.Hooks` and selectively apply some of the non-nil hooks. which works _ok_, as we do hange some hooks in a couple of tests.

but this is not too clean, i'd keep the TODO around for the future